app/src/main/java/org/oppia/app/player/state/hintsandsolution/HintsViewModel.kt

```diff
@@ -0,0 +1,54 @@
+package org.oppia.app.player.state.hintsandsolution
+
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.model.Hint
+import org.oppia.app.model.Solution
+import javax.inject.Inject
+
+/** [ViewModel] for concept card, providing rich text and worked examples */
+@FragmentScope
+class HintsViewModel @Inject constructor(
+) : HintsAndSolutionItemViewModel() {
+
+  var title: String = ""
```

Comment:
Please change all `var` to ObservableField

Reply:
Done.